server/src/sequenceManagement.mjs

```diff
@@ -0,0 +1,58 @@
+/*
+* Copyright 2021 Comcast Cable Communications Management, LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+// sequence management
+
+'use strict';
+
+
+import {sendEvent} from './events.mjs';
+import { getUserIdFromReq } from './util.mjs';
+
+function executeSequence(req,res) {
+  let seqevent
+  const { ws } = res.locals; // Like magic!
```

Comment:
See comment above... this block of code which gets info out of req and res should be in the route function and then this function should just take ws, userId, and seqevent as params (I think; you should confirm these are the params you need/want.)

Reply:
Sure, considered your suggestions and made changes. Please review